### ISSUE TITLE
How to enable only specific date?

### ISSUE THREAD

**Issue by darpanvithani** [fengyuanchen/pickerjs#2]
Hi,

I want to disable all the dates except today and previous four days. How can I achieve this in pickerjs Datepicker?
That means today is 30th Dec and I want to allow user to select Date from 26th Dec. to today.

**Comment by hpolonkoev** [comment]
I don't think this feature is implemented in pockerjs. But it is in his datepicker script : https://github.com/fengyuanchen/datepicker

**Comment by fengyuanchen** [comment]
Not support this for now.

**Comment by itMyron** [comment]
现在特定日期功能实现了吗

**Comment by fengyuanchen** [comment]
@itMyron 暂时没有，没有时间更新，不好意思～
